docs/docs/user-guide/background/megatron_datasets.md

```diff
@@ -1,4 +1,4 @@
-# Writing Megatron-LM compatible datasets
+# Writing Megatron-LM compatible datamodules
```

Comment:
Can you convert this and other headings to title-case? (in this case, just capitalize Compatible Datamodules as well.